_posts/2015-02-17-aerogear-sync.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: AeroGear Sync
```

Comment:
let's use a slightly better blog title (e.g. AeroGear Sync alpha.1 release is out), since that is relfected on the "Hey listen! ..." section of the site